### ISSUE TITLE
[BUG] - Icons are not visible in WHCM if they are in opposite themings

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2407]
### JS Framework

all

### Porsche Design System version

3.0.0-alpha.2

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Reproduction Example

https://designsystem.porsche.com/latest/components/icon/examples

### Steps to reproduce the behavior

1. Go to [PDS Icons on Storefront](https://designsystem.porsche.com/latest/components/icon/examples) on Windows with HC `contrast-black` enabled
2. See error: All icons which are in `light-theme` are invisible
3. Change HC theme to `contrast-white`
4. All icons which arein `theme-light` are now visible
5. Change icon theme to `theme-dark`
6. See error: icons are invisible again

<img width="1389" alt="Bildschirm­foto 2023-03-16 um 09 45 16" src="https://user-images.githubusercontent.com/40788023/225562653-4eacae1c-17bc-47e3-beab-72aeccf61470.png">
<img width="1399" alt="Bildschirm­foto 2023-03-16 um 09 46 34" src="https://user-images.githubusercontent.com/40788023/225562974-b4a56205-7f27-4c00-9d42-684c65ceb364.png">

### Expected behavior

All icons must be visible regardless in which contrast mode they are consumed

**Comment by o-hook** [comment]
duplicate of #2261
